### PR TITLE
add info about VMware NSX-T CNI plugin

### PR DESCRIPTION
**Pull request by egagala**
Hello,

I'm VMware Networking and Security Architect and would like to include short information about our CNI plugin implementation similar to what other vendors did

Best regards

Emil Gagala

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.9 Features: set Milestone to `1.9` and Base Branch to `release-1.9`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.
>
> NOTE: After opening the PR, please *un-check and re-check* the "Allow edits from maintainers" box. This is a temporary workaround to address a known issue with GitHub.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/5987)

### CODE REVIEW

**Comment by egagala** [comment]
I've just signed in CLA agreement

File: docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -190,6 +190,12 @@ Lars Kellogg-Stedman.
...
 
+### NSX-T
+ 
+VMware NSX-T (https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
+ 
+NSX-T Container Plug-in (NCP) (https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS / PaaS platforms such as Pivotal Container Service (PKS), Openshift. 
+
```
Comment (steveperry-53) [inline docs/concepts/cluster-administration/networking.md]: Use square brackets for link text.

[VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html)

[NSX-T Container Plug-in (NCP)[https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf)

Suggestion:
and container-based CaaS/PaaS platforms such as Pivotal Container Service and OpenShift.

Comment (egagala) [reply docs/concepts/cluster-administration/networking.md]: Done according to your suggestions

File: docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -190,6 +190,12 @@ Lars Kellogg-Stedman.
...
 
+### NSX-T
+ 
+[VMware NSX-T] (https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
+ 
+[NSX-T Container Plug-in (NCP)] (https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS / PaaS platforms such as Pivotal Container Service (PKS) and Openshift. 
+
```
Comment (steveperry-53) [inline docs/concepts/cluster-administration/networking.md]: You can see in the [preview](https://deploy-preview-5987--kubernetes-io-master-staging.netlify.com/docs/concepts/cluster-administration/networking/#nsx-t) that these links still aren't working. I think the problem is that you have a space between [...] and (...).

[Working example](https://docs.vmware.com/en/VMware-NSX-T/index.html)

[Not working example] (https://docs.vmware.com/en/VMware-NSX-T/index.html)

Comment (egagala) [reply docs/concepts/cluster-administration/networking.md]: Corrected. I hope now it is good

File: docs/concepts/cluster-administration/networking.md [hunk 3]
```diff
@@ -190,6 +190,12 @@ Lars Kellogg-Stedman.
...
 
+### NSX-T
+ 
+[VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
+ 
+[NSX-T Container Plug-in (NCP)](https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS / PaaS platforms such as Pivotal Container Service (PKS) and Openshift.
```
Comment (xiangpengzhao) [inline docs/concepts/cluster-administration/networking.md]: Please delete the space in `CaaS / PaaS` as @steveperry-53 initially suggested.

Comment (egagala) [reply docs/concepts/cluster-administration/networking.md]: OK. Done. I'll add later to docs/concepts/cluster-administration/addons.md 

I'm new here. Still learning github :)

**Pull request merged**